src/controllers/api.controller.ts

```diff
@@ -24,7 +24,7 @@ export const claimPost = async (
     const userId = (req as ModifiedRequest).userId;
     let rawClaim: any = turnFalsyPropsToUndefined(req.body);
     rawClaim = poormansNormalizer(rawClaim);
-    rawClaim.effectiveDate = new Date(rawClaim.effectiveDate);
+    if (!(rawClaim.effectiveDate instanceof Date)) rawClaim.effectiveDate = new Date(rawClaim.effectiveDate);
```

Comment:
this is going to crash if its not really a date string.  i would put in try-catch please. 

remember candid is not only user of backend, make sure there is some validation or error handling thanks

Reply:
yeah, I actually put that condition because from Candid we are sending the date in a date form ``` effectiveDate = 
 new Date()``` , so the app was crashing, so I made this condition to assure if it's not received as an instance of the Date like what we send in regular creation from the site itself, Convert it